lib/coverband/collectors/base.rb

```diff
@@ -10,7 +10,7 @@ def self.instance
         if Coverband.configuration.collector == 'coverage'
           Thread.current[:coverband_instance] ||= Coverband::Collectors::Coverage.new
         else
-          raise 'select valid collector [trace, coverage]'
+          raise 'select valid collector [coverage]'
```

Comment:
Should we just not have this be configurable if there's only one choice?

Reply:
cleaned up collector base is gone